pip/resolve.py

```diff
@@ -296,12 +308,14 @@ def _resolve_one(self, requirement_set, req_to_install,
                 else:
                     logger.info('Collecting %s', req_to_install)
 
+        assert self.require_hashes is not None
```

Comment:
I had it but the message was too long and it looked dirty to me...

I'll add it.